test/cid-util.spec.js

```diff
@@ -3,17 +3,14 @@
 'use strict'
 
 const { expect } = require('aegir/utils/chai')
+// @ts-ignore
 const multihashing = require('multihashing-async')
 const CID = require('../src')
 const CIDUtil = require('../src/cid-util')
 const uint8ArrayFromString = require('uint8arrays/from-string')
 
-describe('CIDUtil', () => {
-  let hash
-
-  before(async () => {
-    hash = await multihashing(uint8ArrayFromString('abc'), 'sha2-256')
-  })
+describe('CIDUtil', async () => {
```

Comment:
I don't believe `async` for `describe` is supported. It'll work if this can be done within a tick (or so? there's some amount of time that mocha uses to collect all of the tests after running describe()s), but if there's more than a tiny delay then the tests will get entirely dropped from the suite. Doing the setup in an async `before()` is the safest bet.

running with 8.2.1, this runs a single test:

```js
describe('foo', async () => {
  await new Promise((resolve) => setImmediate(resolve))
  it('bar', () => {})
})
```

but this doesn't run any:

```js
describe('foo', async () => {
  await new Promise((resolve) => setTimeout(resolve, 0))
  it('bar', () => {})
})
```

did you also mean to add the `@type` to this one like in index.spec.js?